### PR TITLE
Celery tasks

### PR DESCRIPTION
**Pull request by justinnoah**
blast messages (email, text) as a celery task

### CODE REVIEW

File: _15thnight/__init__.py [hunk 1]
```diff
@@ -11,10 +12,11 @@
 from werkzeug.exceptions import HTTPException
 
 from _15thnight import database
-from _15thnight.database import db_session
-from _15thnight.forms import RegisterForm, LoginForm, AlertForm, ResponseForm, DeleteUserForm
+from _15thnight.forms import (
+    RegisterForm, LoginForm,
+    AlertForm, ResponseForm, DeleteUserForm
```
Comment (nobane) [inline _15thnight/__init__.py]: Could you have this be one line please?

Comment (justinnoah) [inline _15thnight/__init__.py]: It didn't fit and thus my editor yelled at me about it.

Comment (nobane) [inline _15thnight/__init__.py]: There is plenty of line space left before you get close to 79

File: _15thnight/__init__.py [hunk 2]
```diff
@@ -1,5 +1,6 @@
 """15th Night Flask App."""
 
+from celery import Celery
 from email_client import send_email, verify_email
```
Comment (nobane) [inline _15thnight/__init__.py]: I think this is the line that should be removed, not the `send_email` import removed below (leave that one).

File: _15thnight/__init__.py [hunk 3]
```diff
@@ -23,19 +25,26 @@
...
+
 app.secret_key = app.config['SECRET_KEY']
 
 login_manager = LoginManager()
 login_manager.init_app(app)
 login_manager.login_view = 'login'
 
 
+@celery.task
+def blast_messages(email, number, body):
```
Comment (nobane) [inline _15thnight/__init__.py]: Please rename this to `send_alert`.

File: _15thnight/__init__.py [hunk 4]
```diff
@@ -252,17 +274,25 @@ def response_submitted(alert_id):
             body=response_message,
         )
 
-        Response(user=current_user, alert=alert, message=submitted_message).save()
+        Response(
+            Auser=current_user, alert=alert, message=submitted_message
```
Comment (nobane) [inline _15thnight/__init__.py]: Typo here, `Auser`.

File: configdist.py [hunk 5]
```diff
@@ -11,4 +11,6 @@
 EMAIL_SENDER=''
 
 SECRET_KEY = 'This is not secret you must change it'
-HOST_NAME='http://localhost:5000'
\ No newline at end of file
+HOST_NAME='http://localhost:5000'
+
+BROKER = "amqp:///"
```
Comment (nobane) [inline configdist.py]: Can we use sqlite here for the default broker?

File: _15thnight/__init__.py [hunk 6]
```diff
@@ -138,27 +153,31 @@ def dashboard():
...
+            )
             for user in users_to_notify:
                 print("found user to notify {}".format(user))
                 body = "There is a new 15th night alert. Go to " + \
                        HOST_NAME + \
                        "/respond_to/" + \
                        str(alert.id) + " to respond."
-                send_sms(to_number=user.phone_number, body=body)
-                send_email(user.email, '15th Night Alert', body)
+                blast_messages(
```
Comment (nobane) [inline _15thnight/__init__.py]: Actually, let's rename this to `send_alert`.

**Comment by nobane** [comment]
Please squash commits into a single commit.

**Comment by nobane** [comment]
+1

**Pull request merged**